front/poke/temporal/activities.ts

```diff
@@ -59,9 +60,11 @@ import logger from "@app/logger/logger";
 const { DUST_DATA_SOURCES_BUCKET, SERVICE_ACCOUNT } = process.env;
 
 export async function scrubDataSourceActivity({
-  dustAPIProjectId,
+  dataSourceId,
```

Comment:
shouldn't we have the connector deletion here as well?

Reply:
This is handled by the `hardDeleteDataSource` function.